gulpfile.js

```diff
@@ -215,10 +216,10 @@ exports.devHeavy = gulp.series(
   exports.devHeavy
 );
 
-exports['watch-relaunch'] = function() {
-  process.env['BROWSERSYNC_OPEN'] = true;
-  exports.watch();
-}
+// exports['watch-relaunch'] = function() {
```

Comment:
I'd like to see the gulpfiles untouched by this if we can.

Reply:
Done